com.unity.shadergraph/Editor/GraphUI/DataModel/Examples/ShaderGraphExampleTypes.cs

```diff
@@ -196,11 +193,10 @@ public object ObjectValue
             {
                 switch (GetLength())
                 {
-                    case 1: sc(0, (float)value); return;
                     case 2: var v2 = (Vector2)value; sc(0, v2.x); sc(1, v2.y); return;
                     case 3: var v3 = (Vector3)value; sc(0, v3.x); sc(1, v3.y); sc(1, v3.z); return;
                     case 4: var v4 = (Vector4)value; sc(0, v4.x); sc(1, v4.y); sc(1, v4.z); sc(1, v4.w); return;
-                    default: sc(0, 0); return;
+                    default: sc(0, (float)value); return;
```

Comment:
`sc` is so lazy- it's short for "SetComponent". It's a helper for writing a value into CLDS.